packages/design-system/src/components/List/List.a11y.test.tsx

```diff
@@ -3,7 +3,7 @@ import { ROOT_URL } from '@cmsgov/design-system-scripts/helpers/a11y/constants';
 
 import assertNoAxeViolations from '@cmsgov/design-system-scripts/helpers/a11y/assertNoAxeViolations';
 
-const rootURL = `${ROOT_URL}/example/components.list/`;
+const rootURL = `${ROOT_URL}/example/styles.typography.list/`;
```

Comment:
Since List is a class applied to a native list element, do we even need to test the a11y for it? We don't test for other Typography elements.

Reply:
That's a fair point. It might be good to have a check to make sure our example is accessible so we're not encouraging bad practices. I'm going to leave it for now because that decision is probably out of scope for this PR.